etc/ci/report_aggregated_expected_results.py

```diff
@@ -205,7 +206,14 @@ def create_check_run(body: str):
 
 
 def main():
-    results = get_results()
+    parser = argparse.ArgumentParser()
```

Comment:
Instead of adding `--layout-2020` and `--layout-2013` arguments here, maybe we could add a `--tag` argument which is a string type. It could have a default value which would allow using the script more easily from the command-line.

```python
parser.add_argument("--tag", default="", action="store_true",
                                        help="A string tag used to distinguish the results.")
```

Reply:
I used `--layout-2013` and `--layout-2020` to be inlined with other mach tools and make it easier to implelemt it in GitHub actions, but tag would be more futureproof for mac/win wpt runs we could have in the future.